app/ui/cross-sells/src/main/kotlin/com/hedvig/android/crosssells/CrossSells.kt

```diff
@@ -88,30 +85,23 @@ fun CrossSellSheet(
 @Suppress("UnusedReceiverParameter")
 @Composable
 private fun ColumnScope.CrossSellsSheetContent(
-  crossSells: List<CrossSell>,
-  travelAddonBannerInfo: TravelAddonBannerInfo?,
+  recommendedCrossSell: CrossSell?,
+  otherCrossSells: List<CrossSell>,
   onCrossSellClick: (String) -> Unit,
-  onNavigateToAddonPurchaseFlow: (List<String>) -> Unit,
   dismissSheet: () -> Unit,
 ) {
-  HedvigText(stringResource(R.string.CROSS_SELL_TITLE))
-  HedvigText(stringResource(R.string.CROSS_SELL_SUBTITLE), color = HedvigTheme.colorScheme.textSecondary)
-  Spacer(Modifier.height(24.dp))
-  CrossSellsSection(
-    showNotificationBadge = false,
-    crossSells = crossSells,
-    onCrossSellClick = onCrossSellClick,
-    withSubHeader = false,
-  )
-  if (travelAddonBannerInfo != null) {
+  if (recommendedCrossSell != null) {
+    RecommendationSection(recommendedCrossSell, onCrossSellClick)
+  }
+  if (otherCrossSells.isNotEmpty()) {
+    Spacer(Modifier.height(64.dp))
```

Comment:
If the recommended cross sell is in fact null, do we want the 64.dp empty space on top of the rest of the other cross sells content?